daemon/src/engine/audio/Audio.cpp

```diff
@@ -182,6 +183,7 @@ namespace Audio {
     }
 
     void Update() {
+        PROFILE()
```

Comment:
Currently you have to alter the implementation of the functions you are profiling.

I think you should take a look at libunwind and libbacktrace, afaik they allow call stack introspection but I've never used them myself so I can't give you more details.